Gems/ROS2PoseControl/Code/Source/Clients/ROS2PoseControl.h

```diff
@@ -55,23 +55,26 @@ namespace ROS2PoseControl
         //! Obtains the transform in the world tagged with the given tag name.
         //! @param tagName The name of the tag to search for.
         //! @return The transform of the entity with the given tag name, if it exists.
-        AZStd::optional<AZ::Transform> GetOffsetTransform(const AZStd::string& tagName) const;
+        AZStd::optional<AZ::Transform> GetOffsetTransform(const AZStd::string& tagName);
 
         // ImGui::ImGuiUpdateListenerBus::Handler overrides
         void OnImGuiUpdate() override;
 
-        [[nodiscard]] AZ::Outcome<AZ::Transform, const char*> GetCurrentTransformViaTF2() const;
+        [[nodiscard]] AZ::Outcome<AZ::Transform, const char*> GetCurrentTransformViaTF2();
 
         void OnTopicConfigurationChanged();
 
         void OnIsTrackingChanged();
 
         AZ::Transform RemoveTilt(AZ::Transform transform) const;
 
-        AZStd::optional<AZ::Vector3> QueryGround(const AZ::Vector3& location,  const AZ::Vector3& gravityDirection, float maxDistance) const;
+        AZStd::optional<AZ::Vector3> QueryGround(const AZ::Vector3& location,  const AZ::Vector3& gravityDirection, float maxDistance);
```

Comment:
The const parameter was removed but not updated in the source file. This prevents code from compiling.